Tests.Unit.Presentation.Web/Models/V2/ExternalReferenceResponseMapperTest.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Core.DomainModel;
+using Core.DomainModel.GDPR;
+using Core.DomainModel.ItContract;
+using Core.DomainModel.ItSystemUsage;
+using Presentation.Web.Controllers.API.V2.External.Generic;
+using Presentation.Web.Models.API.V2.Response.Shared;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Unit.Presentation.Web.Models.V2
+{
+    public class ExternalReferenceResponseMapperTest : WithAutoFixture
+    {
+        private readonly ExternalReferenceResponseMapper _sut;
+
+        public ExternalReferenceResponseMapperTest()
+        {
+            _sut = new ExternalReferenceResponseMapper();
+        }
+
+        [Fact]
+        public void MapExternalReferenceDtoList_Maps_Contract_References()
+        {
+            //Arrange
+            var contract = new ItContract();
+            AssignExternalReferencesToItContract(contract);
+
+            //Act
+            var dto = _sut.MapExternalReferenceDtoList(contract.ExternalReferences, contract.Reference);
```

Comment:
the differen "contraact.. system .. dpr" tests are identical. You are not using the root for anything, so in that context this is just duplication. Your method takes
- a collection of references
- the master reference

so why this test ended up this way must be due to a shift in direction.